tests/unit/amo/components/TestAddonReviewList.js

```diff
@@ -114,12 +115,23 @@ describe('amo/components/AddonReviewList', () => {
       expect(byLine).toContain(fakeReview.user.name);
     });
 
-    it('renders an icon in the header', () => {
+    it('renders the fallback icon if the origin is not allowed', () => {
```

Comment:
The descriptions of these two tests look like they're swapped, no? This test looks like it expects a real icon URL and the other expects a fallback.